src/Widgets/PageItem.vala

```diff
@@ -37,43 +37,45 @@ public class ENotes.PageItem : Gtk.ListBoxRow {
     private void build_ui () {
         set_activatable (true);
 
+        var margin_horizontal = 10;
```

Comment:
- If we're going to use a constant, you should declare it at the very top of the class as `private const int HORIZONTAL_MARGIN = 10`
- Why the 10? why not something like 6 or 12? (12 is actually the number recommended over at the elementary Human Interface Guidelines: https://elementary.io/docs/human-interface-guidelines#spacing